### ISSUE TITLE
One suggestion regarding grouping element and one question about android native accessibility

### ISSUE THREAD

**Issue by khsbory** [sigute/AccessibilityTipsDemo#1]
Hello, my name is Hyongsop Kim from Korea and work as accessibility QA.
And I'm screen reader user.
The article and demo app source code you wrote really help me study Android accessibility code.
Thank you very much.
While I was looking at the source code regarding grouping element, I got one suggestion and one question.
First is the suggestion.
You said that to improve accessibility navigation, adding contentDescription LinearLayout itself and hide all inside TextView and ImageView with imPortantForAccessibility method is the way to solve the navigation issue with TalkBack.
By the way, as you know, there is a android property called screenReaderFocusable attribute.
I tested within your source code, there is no need importantForAccessibility property by using screenReaderFocusable attribute.
So how about update the code with this attribute?
Next is the one question regarding screenReaderFocusable attribute.
If I add contentDescription itself inside LinearLayout and screenReaderFocusable attribute, it works very well.
But if there is no contentDescription LinearLayout itself and adding just screenReaderFocusable attribute, it doesn't work.
This means that each sub TextViews and ImageView is focusable each even if LinearLayout has screenReaderFocusable.
Do you think is this android accessibility service bug?
Or how to fix this issue with screenReaderFocusable attribute?
I don't want to use android:focusable attribute.
Because if I use this, even if subviews are TextViews, LinearLayout will be focused with keyboard with tab key.

Thank you again for your contribution.
Oh, sorry, I have one more question.
Actually, I have some accessibility solution questions regarding controlling android native app accessibility API.
So may I have your email address?
I want to contact you directly with some questions.
If yes, please send me your email address to hyongsop.kim@nvisions.co.kr.
